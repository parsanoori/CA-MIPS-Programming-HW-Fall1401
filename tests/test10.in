6
237121339
80422940
542248355
1058808998
286349748
666217442