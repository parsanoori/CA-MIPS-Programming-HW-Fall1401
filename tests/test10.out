80422940
237121339
286349748
542248355
666217442
1058808998